source/includes/_users.md

```diff
@@ -0,0 +1,302 @@
+    "groups": [],
+    "active": true,
+    "tenants": ["default_tenant"]
+}
+```
+
+`GET "{manager-ip}/api/v3/users/{user-name}"`
+
+Gets a specific user.
+
+### URI Parameters
+* `user-name`: The name of the user to retrieve.
+
+### Response
+A `User` resource.
+
+
+
+
+
+## List Users
+
+> Request Example
+
+```shell
+$ curl -X GET --header "tenant: default_tenant" -u user:password "http://<manager-ip>/api/v3/users"
+```
+
+```python
+# Python Client-
+client.users.list()
+```
+
+```javascript
+var headers = {
+   'content-type': 'application/json',
+   'authorization': 'Basic ' + new Buffer(username + ':' + password).toString('base64'),
+   'tenant': 'default_tenant'
+}
+
+var settings = {
+  "url": "http://<manager-ip>/api/v3/users",
+  "method": "GET",
+  "headers": headers
+}
+
+$.ajax(settings).done(function (response) {
+  console.log(response);
+});
+```
+
+```html
+obsolete
+```
+
+> Response Example
+
+```json
+{
+ "items":
+    [
+        {
+            "username": "admin",
+            "last_login_at": "2017-01-22T15:09:33.799Z",
+            "role": "administrator",
+            "groups": [],
+            "active": true,
+            "tenants": ["default_tenant"]
+        }
+    ]
+}
+```
+
+`GET "{manager-ip}/api/v3/users"`
+
+List all users.
+
+### Response
+Field | Type | Description
+--------- | ------- | -------
+`items` | list | A list of `User` resources.
+
+
+
+## Create User
+
+> Request Example
+
+```shell
+$ curl -X PUT -H "Content-Type: application/json" -H "tenant: <tenant-name>" -d '{"username": <new-user-name>, "password": <password>, "role": <role>}' -u admin:dbm4uXXPjCuL "http://172.20.0.2/api/v3/users"
+```
+
+```python
+# Python Client-
+client.users.create(<new-user-name>,
+                    <password>,
+                    <role>)
+```
+
+```javascript
+var headers = {
+   'content-type': 'application/json',
+   'authorization': 'Basic ' + new Buffer(username + ':' + password).toString('base64'),
+   'tenant': <tenant-name>
+}
+
+vat data = {
+    "username": <new-user-name>,
+    "password": <password>,
+    "role": <role>
+}
+
+var settings = {
+  "url": "http://<manager-ip>/api/v3/users/<new-tenant-name>",
+  "method": "POST",
+  "headers": headers,
+  "data": data
+}
+
+$.ajax(settings).done(function (response) {
+  console.log(response);
+});
+```
+
+```html
+obsolete
+```
+
+> Response Example
+
+```json
+{
+    "username": "admin",
+    "last_login_at": "2017-01-22T15:09:33.799Z",
+    "role": "administrator",
+    "groups": [],
+    "active": true,
+    "tenants": ["default_tenant"]
+}
+```
+
+`PUT -d '{"username": <new-user-name>, "password": <password>, "role": <role>}' "{manager-ip}/api/v3/users"`
+
+Creates a new user.
+
+### Request Body
+
+Property | Type | Description
+--------- | ------- | -----------
+`username` | string | The user username.
+`password` | string | The user password.
+`role` | string | The user role, one of the following: user, administrator, suspended.
```

Comment:
The user role ~~,~~ **.** **One** ~~one~~ of the following: `user`, `administrator`, `suspended`.